.github/scripts/ci_tests.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set -o errexit # Exit if command failed.
+set -o nounset # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
+echo -e "::group::FreeRTOS Source"
+
+if [ ! -d "libs/FreeRTOS" ]; then
+    git clone https://github.com/FreeRTOS/FreeRTOS.git libs/FreeRTOS
+    pushd libs/FreeRTOS
+    git checkout -b c8fa483b68c6c1149c2a7a8bc1e901b38860ec9b
```

Comment:
Nit1 - consider making this an environment variable at top of file, so it's one-stop shopping when we need to change.


Nit2 - is there a way we could make this "more global" so that every .md/bash/etc. from everywhere referenced the one truth state of this?  I worry this is the type of thing that gets out of sync, where half the files are using 1 commit ID and others are using a different over time. 

Or it breaks laws of physics or becomes even worse to try and "one-truth-source" we should make sure we have clear process for updating this, and as few references of these as we can still do cleanly.

Reply:
Ack. I created global variable for TEST FreeRTOS commit id.

Right now there are two places:
1. Test FreeRTOS
2. Cmake FreeRTOS commit and it can be override by caller (Test is caller of cmake).

Not sure of any better way to do it.

Reply:
I only saw the 2 places too - provided that's it (and i'm hoping it will be) that should be manegaable.  At some point on a future PR you might consider a comment around the commitID that "If you update this, update all of them" just as a reminder to potential future mainteners.  Consider only as i said...